mne/io/base.py

```diff
@@ -1450,6 +1453,54 @@ def save(self, fname, picks=None, tmin=0, tmax=None, buffer_size_sec=None,
                    start, stop, buffer_size, projector, drop_small_buffer,
                    split_size, split_naming, 0, None, overwrite)
 
+    @verbose
+    def export(self, fname, fmt='auto', verbose=None):
+        """Export Raw to external formats.
+
+        Supported formats: EEGLAB (set, uses :mod:`eeglabio`)
+        %(export_warning)s
+        %(export_params_base)s
```

Comment:
I think I'd prefer:
```
%(export_warning)s

Parameters
----------
%(export_params_fname)
```
I know it's less DRY, but it's also clearer that you've entered the Parameters section of the docstring at this point, which is a good tradeoff to me. The first time I read this I had to read extra and stop and think "okay where does Parameters actually start" and then dig into a variable in another file. I think the `docdict` invention is challenging enough for contributors even without the section headers being buried in them :)

Reply:
Makes sense, I was confused by `docdict` at first as well. It'll be nice if it somehow supported formatting.